.builder/jdk.py

```diff
@@ -49,8 +49,10 @@ def install(self, env):
 
         sh = env.shell
 
+        target = '{}-{}'.format(env.spec.target, env.spec.arch)
+
         # If this is a local build, check the local machine
-        if not env.toolchain.cross_compile:
+        if not env.toolchain.cross_compile or target not in URLs:
```

Comment:
do you still need jdk.py here? didn't you just move it into aws-crt-builder?

Reply:
yes, it should have been deleted, will fix